### ISSUE TITLE
Move license file out into separate file

### ISSUE THREAD

**Issue by djmattyg007** [ryanmjacobs/c#9]
Having the license in the same file as the readme makes this more difficult to package than it needs to be., as I would need to manually parse the readme file. If the license was in a separate file, I could just copy it into place.

**Comment by ryanmjacobs** [comment]
What are you trying to build a package for?

**Comment by ellzey** [comment]
primary license should always be its own file. From there you can add any other licenses where you may have derived your code (just make sure it's compatible). For example, libevent/evhtp have something along the lines of 

./LICENSE

```
....
....

Portions of Libevhtp are based on works by others, also made available by them
under the three-clause BSD license above. The functions include:
````

**Comment by ryanmjacobs** [comment]
Closed via e6e83bd.
